crt_portal/cts_forms/templates/forms/complaint_view/index/filter-controls.html

```diff
@@ -4,23 +4,23 @@ <h2 class="intake-section-title">Filters</h2>
   id="filters-form"
   method="GET"
   action="/form/view"
+  label for="filters-form"
```

Comment:
I don't think this works. You most likely want `<label for="...">...</label>` on its own. Curious as to why you added this -- was this an a11y fix?